api/src/main/java/com/epam/pipeline/utils/PipelineUserUtils.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.pipeline.utils;
+
+import com.epam.lifescience.security.entity.UserContext;
+import com.epam.pipeline.entity.user.DefaultRoles;
+import com.epam.pipeline.entity.user.PipelineUser;
+import com.epam.pipeline.entity.user.Role;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.apache.commons.collections4.ListUtils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@SuppressWarnings("HideUtilityClassConstructor")
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class PipelineUserUtils {
```

Comment:
Please, leave a `TODO` note, that some of this logic might be moved directly to corresponding classes (`PipelineUser`, for example).

We can keep the code as-is for now, but let's keep such idea here as well.

Reply:
done